### PR TITLE
Split ca_cert_data into certs and create binding for each

### PR DESCRIPTION
**Pull request by tylerphelan**
- This fixes issues with concatenated certs with ca certs buildpacks

result is multiple secrets in the kpack namespace named and volume + volume mounts for each:
- `ca-certificates-binding-0`
- `ca-certificates-binding-1`

I cannot find a good way to refactor the ytt code

### CODE REVIEW

File: config/ca_cert.yaml [hunk 1]
```diff
@@ -4,17 +4,40 @@
 #@ load("@ytt:data", "data")
 #@ load("@ytt:overlay", "overlay")
 
+#@ def split_certs():
+#@   ca_certs_lines=data.values.ca_cert_data.splitlines()
+#@   ca_certs = []
+#@   i1=0
+#@   i2=0
+#@   for ca_cert_line in ca_certs_lines:
+#@     if ca_cert_line == "-----END CERTIFICATE-----" :
```
Comment (matthewmcnew) [inline config/ca_cert.yaml:13]: Should we error somehow if the cert does not have `-----END CERTIFICATE-----` in it?

File: config/ca_cert.yaml [hunk 2]
```diff
@@ -4,17 +4,40 @@
...
 kind: Secret
 metadata:
-  name: ca-certificates-binding
+  name: #@ "ca-certificates-binding-" + str(i)
   namespace: kpack
 type: servicebinding.io/ca-certificates
 stringData:
   type: ca-certificates
-  certificates: #@ data.values.ca_cert_data
+  certificate: #@ ca_cert
```
Comment (matthewmcnew) [inline config/ca_cert.yaml:36]: Instead of multiple bindings can we just create multiple keys in one binding?

File: config/ca_cert.yaml [hunk 3]
```diff
@@ -4,17 +4,40 @@
...
+
 #@ if data.values.ca_cert_data != "":
+#@ i = 0
+#@ for ca_cert in ca_certs:
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: ca-certificates-binding
+  name: #@ "ca-certificates-binding-" + str(i)
```
Comment (matthewmcnew) [inline config/ca_cert.yaml:31]: Should this be a versioned resource? https://carvel.dev/kapp/docs/v0.46.0/diff/#versioned-resources

File: config/ca_cert.yaml (hunk 2 above)
Comment (tylerphelan) [reply config/ca_cert.yaml]: amazing!

File: config/ca_cert.yaml (hunk 1 above)
Comment (tylerphelan) [reply config/ca_cert.yaml:16]: I added the validation that the last line of `ca_cert_data` must be `-----END CERTIFICATE-----` but it doesn't feel like a super popular error path

File: config/ca_cert.yaml [hunk 4]
```diff
@@ -3,18 +3,46 @@
...
   namespace: kpack
+  annotations:
+    kapp.k14s.io/versioned: ""
 type: servicebinding.io/ca-certificates
 stringData:
   type: ca-certificates
-  certificates: #@ data.values.ca_cert_data
+  #@ i = 0
+  #@ for ca_cert in ca_certs:
+  certificate_(@= str(i) @): #@ ca_cert
```
Comment (matthewmcnew) [inline config/ca_cert.yaml:43]: Nice!

**Review by matthewmcnew — APPROVED** [review]

**Pull request merged**